### ISSUE TITLE
[Design Review] a11y - MHV - Medical Records - Downloadable Reports - Accessibility Wireframe/Prototype Review

### ISSUE THREAD

**Issue by coforma-terry** [department-of-veterans-affairs/va.gov-team#55179]
## Auditor(s)

Sarah Koomson

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [x] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [x] WCAG 2.1 AA
  - [x] WCAG 2.1 AAA
  - [x] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [x] High Contrast Mode
- [x] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav)
  - [x] Annotations for Tab order, reflow and reading order (if needed)
- [x] Accessible Name Annotations
  - [x] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [x] Text alternative for complex images such as charts graphs maps etc.
- [x] Language (ease of reading)
- [x] HTML Tags
  - [x] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [x] [Heading order](https://accessibility.18f.gov/headings/)
   - [x] Annotations (if needed)
  - [x] Semantic Consideration for paragraphs ordered lists numbering
  - [ ] [Tables (heading rows)] (https://design.va.gov/components/table) (Not applicable)
- [ ] Size of clickable areas

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

### Concise issue description

![Image 4-17-23 at 3.26 PM.jpg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/5c063e70-239b-4818-a067-22b19201ce58)

You may also put a PDF icon next to the link to indicate it, but make sure to use a real image, i.e. < img alt ="PDF" /> and not CSS stuff, since the later may not be rendered to screen readers and/or don't give you the opportunity to set alt text (which is very important).
A good practice is also to indicate the file size if its size is big (more than a few megabytes), so that users having a slow or limited connection won't get stuck or burn their mobile data subscription needlessly.
It's also good to indicate the number of pages if it's more than just a few, so that people can have an idea on how big it is, and if they really can take the required time to read it.

PDF Considerations -
Questions from UCD:
Does each page require a H1 (both visually and code)?
Do we need to spell out DOB (Date of birth)?
Should we keep the page numbers in the top right or in the bottom right, or have both?
Do we need a specific file name for the PDF is downloaded - such as including date, patients full name, domain name?

The filename should be irrelevant because the pdf should have a title (meta data).
You don’t need to—and shouldn’t—duplicate headings. The H1 covers the whole document. Just make sure to use a good heading structure ”under” the H1.
”DOB” and most abbreviations should be avoided almost regardless of target audience in my opinion.

## Related Ticket/Epic
[MHV Accessibility Patterns Consolidation for Midpoint #56578](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/56578)

**Comment by coforma-terry** [comment]
Link to wireframes and prototype: [https://www.sketch.com/s/0803544d-e78c-423a-a70e-7db01a9a31c7/p/0816C226-038F-418F-A2CB-883E3951FA6F/canvas]

**Comment by sara-amanda** [comment]
### PDF Reviews Pending
#### Review PDFs (pending Bob per Coulton re: access)
**Per 3/17 Design Intent 3/17:** 
- Fields are a little different - single record if you want to print one flu shot or entire record. 
- This format/template will apply to all records and domains. 
- Make sure that the PDFs are properly marked up and can work with screen readers and follow PDF best practices.  

### PDF Formatting Information
**Per 3/23 MHV Weekly Design Review Meeting:**
- H1 Vaccines
- Individual Vaccines - tried to condense it to save on ink and paper in PDF version (except for provider comments, because those comments have to go below)
- Each Vaccine divided by lines. 
- In this case there is no cover page and no TOC, to save paper and condense. Pulled out only the information that was important, and put it into the header and footer. 
- No color and no watermarks for printing purposes and fax. 

@coultonbunney-usds  @BobbyBaileyRB  @artsymartha68 @coforma-terry

**Comment by sara-amanda** [comment]
### SK PDF Reviews
@coultonbunney-usds  @BobbyBaileyRB , @SarahKay8 has access to Adobe and can assist with the PDF review portion of this ticket, and she is on this ticket, if she can be tagged in anything related to PDFs here, and included in any meetings. Many thanks!

cc: @coforma-terry

**Comment by SarahKay8** [comment]
The filename should be irrelevant because the pdf should have a title (meta data).
You don’t need to—and shouldn’t—duplicate headings. The H1 covers the whole document. Just make sure to use a good heading structure ”under” the H1.
”DOB” and most abbreviations should be avoided almost regardless of target audience in my opinion.

@BobbyBaileyRB 
cc: @sara-amanda

**Comment by SarahKay8** [comment]
Hey @BobbyBaileyRB Design review complete. Please close this out once reviewed.
